src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/SelfRevokeUser/ISelfRevokeOrganizationUserCommand.cs

```diff
@@ -7,7 +9,12 @@
 public interface ISelfRevokeOrganizationUserCommand
 {
     /// <summary>
-    /// Revokes a user from an organization. Validates policy is enabled and user is eligible (not Owner/Admin).
+    /// Revokes a user from an organization.
+    /// Validates the OrganizationDataOwnership policy is enabled and applies to the user (currently Owners/Admins are exempt),
+    /// the user is a confirmed member, and prevents the last owner from revoking themselves.
```

Comment:
I worry about including this much implementation detail in the xmldoc; generally callers don't need to know what checks are done internally. However if you think they're useful to include, I think the `<remarks>` section is better for this.